src/net/sourceforge/kolmafia/session/BreakfastManager.java

```diff
@@ -101,30 +122,20 @@ public static void getBreakfast(final boolean runComplete) {
 
     try (Checkpoint checkpoint = new Checkpoint()) {
       if (runComplete) {
-        checkRumpusRoom();
-        checkVIPLounge();
-        readGuildManual();
-        getHermitClovers();
-        harvestGarden();
-        if (GenericRequest.abortIfInFightOrChoice()) {
-          KoLmafia.updateDisplay(MafiaState.ABORT, "Breakfast aborted.");
-          return;
-        }
-        collectHardwood();
-        useSpinningWheel();
-        visitBigIsland();
-        visitVolcanoIsland();
-        checkJackass();
-        makePocketWishes();
-        haveBoxingDaydream();
-        if (Preferences.getBoolean(
-            "useCrimboToys" + (KoLCharacter.canInteract() ? "Softcore" : "Hardcore"))) {
-          useToys();
+        for (Runnable action : BreakfastManager.ACTIONS) {
+          action.run();
+          if (GenericRequest.abortIfInFightOrChoice()) {
+            KoLmafia.updateDisplay(
+                MafiaState.ABORT, "Breakfast aborted; stuck in fight or choice.");
+            return;
+          }
+          if (StaticEntity.userAborted) {
+            KoLmafia.updateDisplay(MafiaState.ABORT, "Breakfast aborted by user.");
+            return;
+          }
+          ;
```

Comment:
spare semicolon? Kind of surprised spotless didn't catch this.

Reply:
Thanks. Intellij had grayed it out, so I missed it too. odd.